index.js

```diff
@@ -77,16 +101,35 @@ var parseRSS1 = function(xmlObj, callback) {
 var parseRSS2 = function(xmlObj, callback) {
   var json = {feed: {entries: []}};
   var channel = xmlObj.rss.channel[0];
-  if (channel['atom:link']) json.feed.feedUrl = channel['atom:link'][0].href;
+  if (channel['atom:link']) json.feed.feedUrl = channel['atom:link'][0].$.href;
   TOP_FIELDS.forEach(function(f) {
     if (channel[f]) json.feed[f] = channel[f][0];
   })
+  if (channel['itunes:owner']) {
+    json.feed.itunes = {
+      owner: {
+         name: channel['itunes:owner'][0]['itunes:name'][0],
+         email: channel['itunes:owner'][0]['itunes:email'][0]
+      },
+      image: channel['itunes:image'][0].$.href
+    };
+  }
+  PODCAST_TOP_FIELDS.forEach(function(f) {
+    if (channel['itunes:' + f]) json.feed.itunes[f] = channel['itunes:' + f][0];
+  })
   var items = channel.item;
   (items || []).forEach(function(item) {
     var entry = {};
     ITEM_FIELDS.forEach(function(f) {
       if (item[f]) entry[f] = item[f][0];
     })
+    if (item.enclosure) {
+        entry.enclosure = item.enclosure[0].$;
+        entry.itunes = {};
```

Comment:
Let's not create an empty itunes obj unless we know there will be itunes fields. I'd put

```js
entry.itunes = entry.itunes || {}
```
inside the forEach below.